src/App.jsx

```diff
@@ -1,19 +1,96 @@
 import logo from "/logo.png";
 import "./App.css";
+import React, { useEffect } from "react";
+import { Container } from "react-bootstrap";
+import { Row } from "react-bootstrap";
+import { Col } from "react-bootstrap";
+import { getPokemonImage, getRandomPokemon } from "./utils";
+import { useState } from "react";
+import { pokedex } from "./pokedex";
+import { UserGuess } from "./component/userguess";
+import { ScramblePokeName } from "./component/scramblename";
+import { CheckWin } from "./component/checkwin";
+import Button from "react-bootstrap/Button";
 
 function App() {
+  //generating a random pokemon name from the json file.
```

Comment:
this comment is a bit redundant. It is not directly over the function that's being called. And the function name also tells me what is happening